src/lib/setup.js

```diff
@@ -70,6 +71,9 @@ export default function(app) {
     app.use(RavenServer.requestHandler())
   }
 
+  // Minification and compression
+  app.use(compression())
```

Comment:
Wondering if this should be behind a `NODE_ENV === production` condition? 

(Also, related, this is where we gzip our compiled assets: https://github.com/artsy/force/blob/master/scripts/deploy.sh#L7)

Reply:
I was wondering that too, and thought it may be better to keep dev close to prod but I'm not opinionated. Also, this would compress responses Force dynamically generates (but also other stuff since this is global middleware).

@sweir27 @alloy thoughts?

Reply:
Not much.

@damassi Are there in your experience any practical downsides/upsides for one or the other?

Reply:
Nope, not in my experience. Was just thinking that it might require more processing power but if that's negligible then I agree with @yuki24 that we should keep dev and prod closely aligned.